expr/func_groupbytags_test.go

```diff
@@ -0,0 +1,349 @@
+package expr
+
+import (
+	"math"
+	"sort"
+	"strconv"
+	"strings"
+	"testing"
+
+	"github.com/grafana/metrictank/api/models"
+	"github.com/grafana/metrictank/test"
+	schema "gopkg.in/raintank/schema.v1"
+)
+
+func getModel(name string, data []schema.Point) models.Series {
```

Comment:
can't we rely on Series.SetTags here ?

Reply:
Yep. This predated that function, good catch.

Reply:
Done in  448b4d9